guvnor-project/guvnor-project-backend/src/test/java/org/guvnor/common/services/project/backend/server/ProjectRepositoryResolverImplTest.java

```diff
@@ -501,15 +515,17 @@ public void testGetRepositoriesResolvingArtifact_NewGAV_IsInstalledNotDeployed()
 
         } finally {
             if ( oldSettingsXmlPath != null ) {
-                System.setProperty( "kie.maven.settings.custom",
+                System.setProperty( MavenSettings.CUSTOM_SETTINGS_PROPERTY,
                                     oldSettingsXmlPath );
+            } else {
+                System.clearProperty( MavenSettings.CUSTOM_SETTINGS_PROPERTY );
```

Comment:
This (the 6 lines above) seems to be repeating several times in the tests. Would it make sense to move to a method?